internal/keepers/factory.go

```diff
@@ -65,21 +66,40 @@ func (d *keepersReportingFactory) NewReportingPlugin(c types.ReportingPluginConf
 	// number needs to either come from a config, or be calculated on actual
 	// performance of the nodes in real time. that is, start at 1 and increment
 	// after some blocks pass until a stable number is reached.
```

Comment:
nit. TODO can be removed

Reply:
I like this thought, though, on eliminating the need for the second config param. The concept is to achieve the targeted probability while adjusting the number of rounds necessary to achieve it. It's a real-time optimization of upkeep load distribution.